<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMtbRobotDialog</class>
 <widget class="QDialog" name="CMtbRobotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MTB Robot properties dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>MTB robots in scene</string>
   </property>
   <widget class="QListWidget" name="qqRobotList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>381</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Robot Language Program of Selected Robot</string>
   </property>
   <widget class="QPlainTextEdit" name="qqRobotProgram">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqAutomatic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Automatic execution</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
